docs/pages/technology/security_model.rst

```diff
@@ -0,0 +1,87 @@
+- Protecting against eavesdropping and tampering of network traffic between Nuts nodes.
+- Protecting against eavesdropping and tampering of external network traffic between API clients and Nuts node.
+- Protecting against repudiation.
+  Alterations of data and usage of private keys must be written to the audit log.
+- Protecting against denial of service (DoS) attacks from other Nuts nodes.
+  The system should be able to withstand DoS attacks, such as a flood of transactions.
+- Protecting against denial of service (DoS) attacks on public interfaces.
+  The system should be able to withstand DoS attacks, such as a flood of network transactions.
+
+The following are not part of the threat model:
+
+- Protecting against unauthorized database access.
+  If an attacker can inject/modify Nuts node database records (e.g. verifiable credentials), confidentiality of EHR data might be compromised.
+  TODO: should we protect against this? Can we assume an attacker with (Nuts node) database access has access to the EHR database as well?
+- Protecting against arbitrary access to the Nuts node host machine.
+  If an attacker has root access, integrity of configuration is lost and the attacker can alter security settings (e.g. inject authorized API client keys).
+- Protecting against inspection of the Nuts node process.
+  If an attacker can inspect the memory of the Nuts node process, confidentiality might be private keys lost.
+- Protecting against eavesdropping and tampering of internal network traffic between reverse proxy and Nuts node.
+
+External threats
+^^^^^^^^^^^^^^^^
+
+A typical Nuts Node deployment consists of various parts:
+
+* External API clients:
+  * Remote vendor's EHR system
+  * IRMA mobile app
+* Internal API clients:
+  * System administrator using the CLI
+  * Vendor's EHR and administrative system
+  * Monitoring system
+* Reverse proxy for HTTP and gRPC traffic (terminates TLS)
+* Nuts Node
+* Data stores:
+  * Network data
+  * Private key storage
+
+External actors are remote Nuts nodes, remote EHR systems and IRMA mobile devices.
+Remote Nuts nodes and EHR systems require a PKIoverheid TLS client certificate,
+which makes an attack complex: you need to either steal a legal entity's certificate (very hard),
+or buy a certificate for your own legal entity (expensive and time-consuming).
+Then, when the attacker is identified, the certificate can be banned and the legal entity (holder of the certificate) could be held accountable.
+
+The IRMA mobile app is different; it does not get authenticated, so attacks can come from anywhere/anyone.
+Since there are no authentication credentials that can be revoked, attackers can only be stopped by blocking IP addresses or other typical (D)DoS mitigation techniques.
+
+The Nuts node itself does not protect against DoS attacks; the proxy infrastructure routing external traffic to the node will have to protect against this.
```

Comment:
What kind of DoS attacks? Because in the list earlier *transaction flooding* was mentioned. Proxies can't mitigate that specific DoS attack.